tutor/src/models/task-plans/teacher/scores.js

```diff
@@ -385,6 +385,15 @@ class TaskPlanScoresTasking extends BaseModel {
     const wrmQuestions = this.wrmQuestions;
     return some(wrmQuestions, q => q.is_completed);
   }
+
+  @computed get groupQuestionsByPageTopic() {
+    const questions = this.questionsInfo;
+    //order the questions by the exercise page id so when it is grouped, the first chapters are shown first. (assumes page ids are in incremental order)
+    const sortedQuestions = orderBy(questions, ['exercise.page.id'], ['asc']);
```

Comment:
is it possible to order a list using the UUID? I used the id because it was numerical.

Reply:
Ah, I wasn't paying attention to the use of the attribute

Reply:
I do wonder if this could break though... could we get a list of page ordering from the book?

Reply:
True. But I think the pages of the book are in order by id, @nathanstitt is that correct?

Reply:
I don't think we can rely on the order by id.  we do have a `asNumber` method on chapter-section to convert a to a number that can be sorted 

I think this could be:
```
 const sortedQuestions = orderBy(questions, (q) => q.exercise.page.chapt_section.asNumber)

```